source/features/avoid-accidental-submissions.tsx

```diff
@@ -9,32 +10,48 @@ function addQuickSubmit(): void {
 }
 
 function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void {
+	const field = event.delegateTarget;
+	const form = field.form!;
 	if (
 		event.key !== 'Enter'
 		|| event.ctrlKey
 		|| event.metaKey
 		|| event.isComposing // #4323
-		|| select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		|| select.exists([
+			'.suggester', // GitHub’s autocomplete dropdown
+			'.rgh-avoid-accidental-submissions',
+		], form)
 	) {
 		return;
 	}
 
+	if (select.exists('.btn-primary[type="submit"]:disabled', form)) {
+		return;
+	}
+
+	const message = (
+		<p className="rgh-avoid-accidental-submissions my-1">
```

Comment:
This spacing looks great on `isEditingFile`:

<img width="726" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/125909257-13e605e6-c7c5-4c25-b00f-c611184b9d95.png">

But not so much on `isNewIssue`:

<img width="733" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/1402241/125909361-7a2b2ec1-5773-4ff4-bfa1-28db4bae3f1c.png">

`mt-2 mb-0` would work better:

<img width="726" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/125908555-c02f51a0-dd8d-4111-a10e-6640326cf1b9.png">

Let's also make sure the spacing is right in every page this feature runs on. Can you fix this and post some screenshots?

Reply:
Yes, I have fixed this following your suggestion.
Here are screenshots:

<details>
<summary>New Issue</summary>

![2021-07-16_170625](https://user-images.githubusercontent.com/723651/125960739-c62c1652-463d-4011-bac4-1ecf420b5428.jpg)
</details>

<details>
<summary>New PR/branch compare</summary>

![2021-07-16_170555](https://user-images.githubusercontent.com/723651/125960670-58982a13-0b58-4ded-afbf-7408db20f093.jpg)
</details>

<details>
<summary>PR merge</summary>

![2021-07-16_170241](https://user-images.githubusercontent.com/723651/125960255-ee4b1267-e5c2-47eb-b561-14cd50bf34a7.jpg)
</details>

<details>
<summary>New file commit</summary>

![2021-07-16_163631](https://user-images.githubusercontent.com/723651/125956756-2d05f180-ef75-4202-ac78-f2d541ebb757.jpg)
</details>

<details>
<summary>Edit file commit</summary>

![2021-07-16_163706](https://user-images.githubusercontent.com/723651/125956761-cec9f75b-7e9b-4dd6-b462-198358782857.jpg)
</details>